### DISCUSSION TITLE
How to get Surface Normals

### DISCUSSION THREAD

**Discussion by Reuver** [bnpr/Malt#493] [Q&A]
In the old 1.0 video (https://www.youtube.com/watch?v=HiGYx_DADOQ), Miguel is able to get the surface normals using a node called surface normals. 

That node is gone now, but I guessed a geometry node which has a screen space setting and a normal output, would give them, so I added one to a Screen Pass, and it gave a black output.
Then I saw that the PrePass node has a normal depth output, which also gave a black output.  Am I doing something wrong? 

Edit - I solved it, I forgot I had to add the Screen Pass to the Render Layer. It looks like GitHub doesn't allow these to be removed by me, so I apologize for this.

**Comment by eballeste** [comment]
No need to apologize, if anything i would separate your edit into it's own response and mark it as the answer (notice the top status of unanswered). 

hopefully it can help other people who are looking to solve the very same thing.

maybe also add a screenshot for visual reference? that would elevate the helpfulness levels to star levels.
